src/main/java/sirius/kernel/nls/NLS.java

```diff
@@ -185,23 +185,21 @@ public static void setDefaultLanguage(String lang) {
      * By default, SIRIUS initializes with the language set in {@code nls.defaultLanguage} so a switchover
      * to the system language has to be performed manually.
      *
-     * @return the language code of the underlying operating system. If the language is not supported (not listed
-     * in {@code nls.languages}), <tt>null</tt> will be returned as
-     * {@link sirius.kernel.async.CallContext#setLang(String)} doesn't change the current language if <tt>null</tt> is
-     * passed in.
+     * @return the language code of the underlying operating system
      */
     @Nullable
     public static String getSystemLanguage() {
-        return makeLang(Locale.getDefault().getLanguage().toLowerCase());
+        return Locale.getDefault().getLanguage().toLowerCase();
```

Comment:
Why the functionality change here?

Reply:
makeLang ist deprecated forRemoval - this code almost never runs anyway, only in case of a completely empty system config..

Reply:
k